apps/omg/lib/omg/input_pointer.ex

```diff
@@ -14,33 +14,135 @@
 
 defmodule OMG.InputPointer do
   @moduledoc """
-  `OMG.InputPointer` and `OMG.InputPointer.Protocol` represent the data that's used to mention outputs that are intended
-  to be inputs to a transaction. Examples are UTXO positions or output id (consisting of transaction hash and output
-  index)
+  `OMG.InputPointer` represent the data that's used to mention outputs that are intended 
+  to be inputs to a transaction. Examples are UTXO positions or output id.
+  (consisting of transaction hash and output index)
+  """
+
+  require OMG.Utxo
+
+  # TODO(achiurizo)
+  # Do we really need this? How does this work with output type marker?
+  @input_pointer_type_marker <<1>>
+
+  @type utxo_pos_tuple() :: {
+          :utxo_position,
+          # blknum
+          non_neg_integer,
+          # txindex
+          non_neg_integer,
+          # oindex
+          non_neg_integer
+        }
+
+  # This is the 'legacy' db format that the code is using to maintain backwards
```

Comment:
no reason to keep this comment I presume. or rephrase it so it makes more sense :-)